cli/command/bundlefile/bundlefile_test.go

```diff
@@ -37,7 +38,7 @@ func TestLoadFileSyntaxError(t *testing.T) {
 	}`)
 
 	_, err := LoadFile(reader)
-	assert.EqualError(t, err, "JSON syntax error at byte 37: invalid character 'u' looking for beginning of value")
+	assert.Check(t, is.Error(err, "JSON syntax error at byte 37: invalid character 'u' looking for beginning of value"))
```

Comment:
Is it possible to automate changing these to `assert.Error()` (https://github.com/gotestyourself/gotestyourself/pull/71), or do you want to keep that for a follow-up ?

Reply:
It's possible to automate the change, but it's not the same as gotestyourself/gotestyourself#71.

The current automated migration preserves behaviour, so `testify/require` -> `Assert`, `testify/assert` -> `Check`. 

I can write another migration that converts specific checks to the canonical `Assert`, but that is a change in behaviour (Fail -> FailNow), so I'd like to do that in a separate PR so that we can more carefully review the change.